catalog/spatial/registry/registry-publication-action-provider/src/main/java/org/codice/ddf/registry/publication/manager/RegistryPublicationManager.java

```diff
@@ -111,25 +95,18 @@ public void init() throws FederationAdminException, PrivilegedActionException {
         // To here
 
         for (Metacard metacard : metacards) {
-            Attribute registryIdAttribute =
-                    metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID);
-            if (registryIdAttribute == null || StringUtils.isBlank(registryIdAttribute.getValue()
-                    .toString())) {
+            if (!RegistryUtility.hasRegistryId(metacard)) {
```

Comment:
Can we getRegistryId here and check for null and not have to getRegistryId below?